acceptance/suites/pre_suite/puppet4_compat/70_install_puppet.rb

```diff
@@ -0,0 +1,10 @@
+require 'puppetserver/acceptance/compat_utils'
+
+step "Install Legacy Puppet Agents."
+
+puppet_version = ENV['PUPPET_LEGACY_VERSION']
+if not puppet_version
+  fail "PUPPET_LEGACY_VERSION is not set, e.g. '1.9.3'"
+end
+
+install_puppet_agent_on(nonmaster_agents, {:puppet_agent_version => puppet_agent_version})
```

Comment:
Oi. Fixed this, and ran the Puppet 3 and Puppet 4 compat tests. We should be good this time.